package.json

```diff
@@ -29,7 +29,9 @@
     "js-yaml": "^4.1.0",
     "markdown-to-jsx": "^7.1.6",
     "moment": "2.29.4",
+    "moment-timezone": "^0.5.45",
     "next": "14.1.1",
+    "next-plausible": "^3.11.1",
```

Comment:
I think we have removed this from our project recently `"next-plausible": "^3.11.1",` .

Related PR : https://github.com/json-schema-org/website/pull/737

Please make sure you are in latest upstream.

Reply:
updated to upstream - 6bc564e0d91041a1e3f54babf2a87cbbe100ae61